staging/src/k8s.io/apiserver/pkg/apis/audit/validation/validation.go

```diff
@@ -60,3 +65,38 @@ func validateLevel(level audit.Level, fldPath *field.Path) field.ErrorList {
 		return field.ErrorList{field.NotSupported(fldPath, level, validLevels)}
 	}
 }
+
+func validateNonResourceURLs(urls []string, fldPath *field.Path) field.ErrorList {
```

Comment:
liggitt saies that RBAC doesn't do validation in kube-apiserver.
But I fount this:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/audit/policy/checker.go#L112